build/patches/Add-DuckDuckGo-Lite-search-engine.patch

```diff
@@ -31,11 +31,11 @@ diff --git a/components/search_engines/prepopulated_engines.json b/components/se
 diff --git a/components/search_engines/search_engine_type.h b/components/search_engines/search_engine_type.h
 --- a/components/search_engines/search_engine_type.h
 +++ b/components/search_engines/search_engine_type.h
-@@ -76,6 +76,7 @@ enum SearchEngineType {
-   SEARCH_ENGINE_ECOSIA = 59,
-   SEARCH_ENGINE_PETALSEARCH = 60,
-   SEARCH_ENGINE_GOOGLE_EN = 61,
-+  SEARCH_ENGINE_DUCKDUCKGOLIGHT = 62,
+@@ -78,6 +78,7 @@ enum SearchEngineType {
+   SEARCH_ENGINE_STARTER_PACK_BOOKMARKS = 61,
+   SEARCH_ENGINE_STARTER_PACK_HISTORY = 62,
+   SEARCH_ENGINE_GOOGLE_EN = 63,
++  SEARCH_ENGINE_DUCKDUCKGOLIGHT = 64,
```

Comment:
I don't know if re-numbering is ok

Reply:
Yes, there's no problem.
Moreover, even if an item is missing, nothing changes to the user, as the url used for the search is stored in the preferences. It just won't appear in the selection list (but only after a storage reset).

Reply:
There is a problem: new engines will not appear if the latest number did not change. It is explained in the comments there.